test/unit/math/rev/mat/prob/normal_id_glm_lpdf_test.cpp

```diff
@@ -0,0 +1,162 @@
+#include <stan/math/rev/mat.hpp>
+#include <gtest/gtest.h>
+#include <stan/math/prim/scal/fun/value_of.hpp>
+
+using stan::math::var;
+using Eigen::Dynamic;
+using Eigen::Matrix;
+
+//  We check that the values of the new regression match those of one built
+//  from existing primitives.
+TEST(ProbDistributionsNormalIdGLM, glm_matches_normal_id_doubles) {
+  Matrix<int, Dynamic, 1> n(3, 1);
+  n << 51, 32, 12;
+  Matrix<double, Dynamic, Dynamic> x(3, 2);
+  x << -12, 46, -42,
+      24, 25, 27;
+  Matrix<double, Dynamic, 1> beta(2, 1);
+  beta << 0.3, 2;
+  double alpha = 0.3;
+  Matrix<double, Dynamic, 1> alphavec = alpha * Matrix<double, 3, 1>::Ones();
+  Matrix<double, Dynamic, 1> theta(3, 1);
+  theta = x * beta + alphavec;
+  Matrix<double, Dynamic, 1> sigma(3, 1);
+  sigma << 10, 4, 6;
+
+  EXPECT_FLOAT_EQ((stan::math::normal_lpdf(n, theta, sigma)),
```

Comment:
Out of curiosity, why do you do one without the `propto` specified?

Reply:
This is a great question. Of course, that does the same thing as with `propto=false`. I copied this from other existing tests and figured that seeing that they found that worth testing, I should probably test it as well.